### PR TITLE
chore(coveralls): move from posttest to a separate script for npm and travis

### PR DESCRIPTION
**Pull request by prtksxna**

### CODE REVIEW

**Comment by sanjayaksaxena** [comment]
Shouldn't there be one more script for running coverage locally using **istanbul**?

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "pretest": "npm run lint && npm run hint",
     "test": "./node_modules/.bin/_mocha ./test/",
-    "posttest": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
+    "coveralls": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
```
Comment (sanjayaksaxena) [inline package.json]: I guess we will need another script for local coverage test...

Comment (sanjayaksaxena) [inline package.json]: If we restore the old test & coverage script under a different name and also make **it do linting** then it should suffice... this was the old script:

`./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha ./test/`

**Comment by prtksxna** [comment]
The `test` script will now run mocha, istanbul, and are linting and hinting. Travis will run the `coveralls` script separately.

File: package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "pretest": "npm run lint && npm run hint",
```
Comment (sanjayaksaxena) [inline package.json]: cool :)

**Pull request merged**